bundles/managed-api-service/1.20.0/manifests/managed-api-service.clusterserviceversion.yaml

```diff
@@ -51,15 +51,16 @@ metadata:
+        {"component_name":"3scale-operator.v0.8.3","component_url":"registry.redhat.io/3scale-amp2/3scale-rhel7-operator@sha256:06878a946d0011bcf45c35d6454e64054a7d7b54444edfc2af3769b95eb0c218"},{"component_name":"backend","component_url":"registry.redhat.io/3scale-amp2/backend-rhel8@sha256:1662c92f37a9e54a210f2022027c312b7d778d2481191fe020d5fe40d1adf39d"},{"component_name":"apicast","component_url":"registry.redhat.io/3scale-amp2/apicast-gateway-rhel8@sha256:14c01cbc43b3756850fa9732d9a2f588792ec375d6834866f04a28fbc6c04738"},{"component_name":"system","component_url":"registry.redhat.io/3scale-amp2/system-rhel7@sha256:6834ddaa12ca1e15d2b96c3ae0e0fc4e890fe38ff48f6543d27143528d7aadbc"},{"component_name":"zync","component_url":"registry.redhat.io/3scale-amp2/zync-rhel8@sha256:62b24bde0123a22241e02e6f7b9c7a55794e586b852420e587cc034a8cf76e9e"},{"component_name":"system_memcached","component_url":"registry.redhat.io/3scale-amp2/memcached-rhel7@sha256:f31e07c6ec15bb823b40af5bd8c96bb0083f9c8cbcc643c43f1854e7002e480a"},{"component_name":"cloud-resources.v0.38.0","component_url":"quay.io/integreatly/cloud-resource-operator:v0.38.0"},{"component_name":"grafana-operator.v3.10.4","component_url":"quay.io/rhoas/grafana-operator:v3.10.4"},{"component_name":"marin3r.v0.10.0","component_url":"quay.io/3scale/marin3r:v0.10.0"},{"component_name":"keycloak-operator.v17.0.0","component_url":"quay.io/keycloak/keycloak-operator:17.0.0"},{"component_name":"observability-operator.v3.0.8","component_url":"quay.io/rhoas/observability-operator:v3.0.8"}]
```

Comment:
@laurafitzgerald looks like the sso related images are not present

Reply:
@MStokluska do you know where these are picked up from? We removed some hard coded places where these images were coming from and used only the SSO CSV for these. Could that have changed something about the containerImage logic?

Reply:
@trepel going to hold off on running the rest of the release pipeline till we get to the bottom of this

Reply:
Hey @laurafitzgerald @austincunningham reason why they are not listed in the containerImages is because Keycloak Operator 17.0.0 does not include them in the CSV, 15.02 did. Denis is doing some work around getting these in a seperate config file. Alternative to this would be to reach out to kc team to ask about why are the related images gone, but I don't think it is necessary and we are good to go here!

Reply:
I think the productized CSV should contain the related images. i.e. if you look at 15.02 on github it also doesn't have related images in the CSV https://github.com/keycloak/keycloak-operator/blob/main/deploy/olm-catalog/keycloak-operator/15.0.2/manifests/keycloak-operator.v15.0.2.clusterserviceversion.yaml but the 15.02 does in our last release https://github.com/integr8ly/integreatly-operator/blob/rhoam-release-v1.19/manifests/integreatly-rhsso/15.0.2/keycloak-operator.v15.0.2.clusterserviceversion.yaml#L197-L204 @adam-cattermole I am wondering do we have the right Bundle

Reply:
looking at older releases keycloak in rhoam 14.0, 13.0 we don't have related images in it think we can plow ahead

Reply:
If I am remembering correctly I think 15.02 was the first time we used the productized bundles. I think we should have a look for rc2 to see if we can get the productized bundle for keycloak. This shouldn't block RC1 though

Reply:
> I think the productized CSV should contain the related images. i.e. if you look at 15.02 on github it also doesn't have related images in the CSV https://github.com/keycloak/keycloak-operator/blob/main/deploy/olm-catalog/keycloak-operator/15.0.2/manifests/keycloak-operator.v15.0.2.clusterserviceversion.yaml but the 15.02 does in our last release https://github.com/integr8ly/integreatly-operator/blob/rhoam-release-v1.19/manifests/integreatly-rhsso/15.0.2/keycloak-operator.v15.0.2.clusterserviceversion.yaml#L197-L204 @adam-cattermole I am wondering do we have the right Bundle

My understanding was that these related images were added in 15.0.2 so that we could override the version of SSO deployed by keycloak operator to a newer version as the keycloak image was yet to be updated.